src/chain/electrum.rs

```diff
@@ -130,7 +130,7 @@ impl ElectrumRuntimeClient {
 				BDK_ELECTRUM_CLIENT_BATCH_SIZE,
 				true,
 			)
-		});
+		})?;
```

Comment:
Can't this just panic? Not starting the runtime is such a high level problem, that I'd think that panic is fine?

Reply:
It's not not being able to start the runtime, but the question mark is only needed when start wasn't called at all?